docker/Dockerfile_ppp_openmc

```diff
@@ -194,38 +237,38 @@ RUN cd $HOME && git clone https://github.com/ukaea/parallel-preprocessor.git &&
     make install && \
     cd .. && rm -rf build
 
-##################  nuclear material data for openmc #########################
-# install endf nuclear data, this could be skipped and use volume map instead
-# must mapped to /mat_dir
+##################  
+## nuclear material data for openmc 
+###################
+# consider to split the docker image from here
+# copy data into docker image could be skipped and use volume map instead
+# but host folder must be mapped to /mat_dir in the container
```

Comment:
There are two options given here, and what is actually being done is not completely clear.  The documentation seems to suggest that the data _is_ currently being included in the Docker file.  In the readme file:

> Since nuclear material (tendl-2019-hdf5) has been built in, there is no need to map host folder to container like this

However, it looks like the material is _not_ currently being built in - the line cloning the data repository is commented out below.